lokerpc/codegen/testdata/spaces-hyphens.json.ts

```diff
@@ -2,9 +2,10 @@ import { RPCContextClient } from "@loke/http-rpc-client";
 import { Context } from "@loke/context";
 
 export type User = {
-  "First Name": string;
+  "Email@Address": string;
+  "First-Name": string;
   "Last Name": string;
-  "User ID": string;
+  "User_ID": string;
```

Comment:
Hmm underscore should be fine, did I get the regex wrong?

Also a case with a quote in it is probably a good edge case

Reply:
Ah you are correct, shouldn't need to quote this case. Will check the regexp.

Reply:
Yeah, regex is def NQR

```

export type User = {
  "\"DoubleQuotes\"": string;
  "#prefixed": string;
  "'SingleQuotes'": string;
  "@prefixed": string;
  "Kebab-Case": string;
  "PascalCase": string;
  "Snake_Case": string;
  "camelCase": string;
  "email@address.com": string;
  "kebab-case": string;
  snake_case: string;
};
```

Reply:
Oh first letter can be uppercase

Reply:
https://stackoverflow.com/questions/15326421/how-do-i-do-a-case-insensitive-regular-expression-in-go

Reply:
Yeah case sensitivity fixed it. Could have probably also just included A-Z

```
export type User = {
  "\"DoubleQuotes\"": string;
  "#prefixed": string;
  $prefixed: string;
  "'SingleQuotes'": string;
  ".prefixed": string;
  "1numberedStart": string;
  "@prefixed": string;
  "Kebab-Case": string;
  PascalCase: string;
  Snake_Case: string;
  camelCase: string;
  "comma,separated": string;
  "dot.separated": string;
  "email@address": string;
  "kebab-case": string;
  lowercase: string;
  numberedEnd1: string;
  "semicolonsuffixed;": string;
  snake_case: string;
};
```